### PR TITLE
Adds some full Th'akh Wrappings

### PR DESCRIPTION
**Pull request by Geevies**
It's Canon's custom item, whom gave me permission to add it to the general loadout.

image

### CODE REVIEW

**Comment by Geevies** [comment]
!review

File: code/modules/clothing/under/xenos/unathi.dm [hunk 1]
```diff
@@ -107,4 +107,26 @@
...
+/obj/item/clothing/suit/unathi/mantle/wrapping
+	name = "unathi wrappings"
+	desc = "Stitched together clothing with bandages covering them, looks tailored for an unathi."
+	description_fluff = "This is a very traditional Sinta wear for Th'akh shamans, most modern Sinta do not wear these."
+	icon = 'icons/obj/custom_items/karnaikai_wrappings.dmi'
+	icon_state = "karnaikai_wrappings" //special thanks to Araskael
+	item_state = "karnaikai_wrappings"
+	body_parts_covered = UPPER_TORSO|LOWER_TORSO|LEGS|ARMS
+	flags_inv = HIDEJUMPSUIT|HIDETAIL
+	species_restricted = list("Unathi")
```
Comment (NonQueueingMatt) [inline code/modules/clothing/under/xenos/unathi.dm]: So the autakh can get them in the loadout but not wear them?

File: code/modules/client/preference_setup/loadout/loadout_xeno/unathi.dm [hunk 2]
```diff
@@ -120,3 +120,19 @@
 	path = /obj/item/clothing/under/unathi/zozo
 	whitelisted = list("Unathi", "Aut'akh Unathi")
 	sort_category = "Xenowear - Unathi"
+
+/datum/gear/suit/unathi/wrapping_head
+	display_name = "thakh shaman head wrappings"
+	description = "Head wrappings with a breath mask. Only very traditional Th'akh Shamans would wear these."
+	path = /obj/item/clothing/mask/gas/wrapping
+	cost = 1
+	whitelisted = list("Unathi", "Aut'akh Unathi")
```
Comment (Geevies) [inline code/modules/client/preference_setup/loadout/loadout_xeno/unathi.dm]:
```suggestion
	whitelisted = list("Unathi")
```

File: code/modules/client/preference_setup/loadout/loadout_xeno/unathi.dm [hunk 3]
```diff
@@ -120,3 +120,19 @@
...
+	sort_category = "Xenowear - Unathi"
+
+/datum/gear/suit/unathi/wrapping_body
+	display_name = "thakh shaman body wrappings"
+	description = "Closed body wrappings. Only very traditional Th'akh Shamans would wear these."
+	path = /obj/item/clothing/suit/unathi/mantle/wrapping
+	cost = 1
+	whitelisted = list("Unathi", "Aut'akh Unathi")
```
Comment (Geevies) [inline code/modules/client/preference_setup/loadout/loadout_xeno/unathi.dm]:
```suggestion
	whitelisted = list("Unathi")
```

File: code/modules/clothing/under/xenos/unathi.dm [hunk 4]
```diff
@@ -107,4 +107,26 @@
...
 		rolled_sleeves = TRUE
-	update_clothing_icon()
\ No newline at end of file
+	update_clothing_icon()
+
+/obj/item/clothing/suit/unathi/mantle/wrapping
+	name = "unathi wrappings"
+	desc = "Stitched together clothing with bandages covering them, looks tailored for an unathi."
+	description_fluff = "This is a very traditional Sinta wear for Th'akh shamans, most modern Sinta do not wear these."
+	icon = 'icons/obj/custom_items/karnaikai_wrappings.dmi'
```
Comment (Arrow768) [inline code/modules/clothing/under/xenos/unathi.dm]: If its not a custom item anymore, change the icon path aswell and remove the custom item, unless there is a difference to the one added here

**Comment by ParadoxSpace** [comment]
Besides Arrow's thing, looks good.

**Pull request merged**